### PR TITLE
Override build OpenMRS frontend script & use core version `5.6.1-pre.1857`

### PR DESCRIPTION
**Pull request by corneliouzbett**
This PR Overrides OpenMRS frontend core version with `5.6.1-pre.1857`

### CODE REVIEW

File: base/configs/openmrs/frontend_assembly/spa-assemble-config.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
+    "coreVersion": "5.6.1-pre.1857",
     "frontendModules": {
         "@openmrs/esm-form-entry-app": "7.1.0"
```
Comment (vasharma05) [inline base/configs/openmrs/frontend_assembly/spa-assemble-config.json:4]: @corneliouzbett , please pull in the latest changes, the version for the `@openmrs/esm-form-entry` needs to be `8.0.0`.
Thanks!

File: base/pom.xml [hunk 2]
```diff
@@ -175,6 +175,29 @@
               </resources>
             </configuration>
           </execution>
+
+          <execution>
+            <id>Override build OpenMRS Frontend Script</id>
```
Comment (rbuisson) [inline base/pom.xml:180]: Is this actually overriding a build step from Maven Parent? In which case the `id` is important and cannot be changed.

If not, then let's rename this to:
```suggestion
            <!-- Override the Groovy script provided by Ozone to allow overriding the App Shell version -->
            <id>Override OpenMRS frontend build Groovy script</id>
```

File: prod/pom.xml [hunk 3]
```diff
@@ -63,6 +63,35 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
+      <groupId>org.apache.maven.plugins</groupId>
+      <artifactId>maven-resources-plugin</artifactId>
+      <executions>
+        <execution>
+          <id>Override build OpenMRS Frontend Script</id>
```
Comment (rbuisson) [inline prod/pom.xml:72]: Same as above.

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: base/pom.xml (hunk 2 above)
Comment (corneliouzbett) [reply base/pom.xml]: > Is this actually overriding a build step from Maven Parent? In which case the id is important and cannot be changed.

Just overriding a file.

**Review by rbuisson — APPROVED** [review]

**Pull request merged**